docs/reference/glossary.md

```diff
@@ -378,7 +393,11 @@ See [Resource accounts](../move/move-on-aptos/resource-accounts.md) for instruct
 ### Sender
 
 - _Alternate name_: Sender address.
-- **Sender** is the address of the originator account for a transaction. A transaction must be signed by the originator.
+- **Sender** is the address of the originator account for a transaction. A transaction must be signed by the originator but can have more than one signer.
```

Comment:
This is confusing as a whole tbh. And I tried to rewrite it below but I don't like it either.

**Sender** is the originator account for a transaction.  A transaction must be signed by the sender, but can be signed by others as well.